code/style.css

```diff
@@ -1,7 +1,487 @@
+  display: flex;
+}
+
+.header__video--ipad {
+  display: none;
+}
+
+.header__video--desktop {
+  display: none;
+}
+
+.navbar {
+  width: 100%;
+  height: 80px;
+  background-color: rgb(0, 0, 0);
+  color: white;
+  display: grid;
+  grid-template-columns: 1fr auto auto auto;
+  align-items: center;
+  padding: 0 3%;
+  column-gap: 10px;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+}
+
+.navbar__logotype {
+  height: 30px;
+  width: 200px;
+}
+
+.navbar__item {
+  text-align: center;
+  display: grid;
+  grid-template-rows: auto auto;
+  row-gap: 10px;
+  text-decoration: none;
+  color: white;
+  font-size: 14px;
+}
+
+.navbar__item:hover {
+  color: rgb(255, 147, 6);
+  transition: color 1s ease-in;
+  border-bottom: 1px solid rgb(255, 147, 6);
+}
+
+.navbar__item a:active {
+  transition: all 1s ease-in;
+  color: rgb(255, 147, 6);
+  border-bottom: 1px solid rgb(255, 147, 6);
+}
+
+.form-accordion-contact-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.hidden-anchor {
+  display: block;
+  position: relative;
+  top: -80px;
+  visibility: hidden;
+}
+
+/*------------- FORM CONTAINER ---------------- */
+
+.form {
+  padding: 30px 0;
+  display: grid;
+  max-width: 1200px;
+  margin: auto;
+}
+
+.form__content {
+  padding: 0 10%;
+}
+
+h2 {
+  margin-bottom: 20px;
+  padding: 0;
+  width: 100%;
+}
+
+.form__content__layout {
+  display: grid;
+  row-gap: 10px;
+}
+
+.form__content__layout__row {
+  font-size: 16px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.form__content__layout__row__input-text {
+  padding: 5px 5px 5px;
+  -webkit-appearance: none;
+  background-color: rgb(255, 254, 253, 0);
+  border: 0;
+  border-bottom: 1px solid gray;
+  font-size: 16px;
+  background-color: white;
+}
+
+.form__content__layout__row__input-text:focus,
+.form__content__layout__row__room-selector:focus,
+.form__content__layout__row__date-picker:focus {
+  outline: none;
+  border: 1px solid rgb(255, 147, 6);
+  transition: all 1s ease-in;
+}
+
+input:focus::placeholder {
+  color: white;
+  transition: all 1s ease-in;
+}
+
+.form__content__layout__row__room-selector {
+  font-size: 15px;
+  padding: 5px 5px 5px;
+  outline: none;
+}
+
+.form__content__layout__row__date-picker {
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+  outline: none;
+  padding: 5px 5px 5px;
+}
+
+.form__content__layout__row--radio-btns {
+  display: grid;
+  grid-template-columns: 1fr auto;
+}
+
+.form__content__layout__row__radio-btn-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  column-gap: 10px;
+  row-gap: 4px;
+}
+
+input[type="radio"] {
+  -webkit-appearance: none;
+  display: block;
+  width: 1em;
+  height: 1em;
+  border-radius: 50%;
+  border: 0.1em solid black;
+  margin: auto;
+  box-shadow: 1px 1px black;
+}
+
+input[type="radio"]:checked {
+  background-color: black;
+  box-shadow: none;
+  transition: all 0.5s ease-in;
+}
```

Comment:
Loved the styling in the radio buttons!! The transitions made it look super nice.